README.md

```diff
@@ -55,6 +55,31 @@ The example apps are supporting two kinds of configuration: `authenticationCloud
 To change the configuration open the [AppAssembly.swift](NevisExampleApp/Dependency%20Provider/AppAssembly.swift) file which describes the dependency injection related configuration using the `Swinject` library.
 The `environment` parameter should be changed when injecting the `ConfigurationLoaderImpl` component to one of the values already mentioned.
 
+#### Handling deep links
+
+The example applications handle deep links which contain a valid dispatchTokenResponse query parameter of an out-of-band operation.
+
+The feature is achieved with Custom URL Schemes.
+
+> [!NOTE]
+> Further information: [Define custom url scheme](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app).
+
+##### Custom URL Schemes
+
+Modify the content of `CFBundleURLSchemes` array in the [Info.plist](NevisExampleApp/Resources/Info.plist) file with the right scheme information of your environment.
+
+```plist
```

Comment:
Is `plist ` a valid language identifier? If you take a look at to the rendered [README](https://github.com/nevissecurity/nevis-mobile-authentication-sdk-example-app-ios/tree/feature/NEVISACCESSAPP-5263-add-deep-link-usage-description) it looks weird to me.